src/CustomMimetypeHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Http\Message\MultipartStream;
+
+/**
+ * Let you add your own mimetypes.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CustomMimetypeHelper extends ApacheMimetypeHelper
+{
+    /**
+     * @var array
+     */
+    private $mimetypes = [];
+
+    /**
+     * @param array $mimetypes
```

Comment:
i am a fan of stating the kindof obvious but maybe not really... i think we should say here that this is a hashmap of extension => mime type